site/content/docs/commands/storage-init.md

```diff
@@ -24,6 +24,11 @@ DynamoDB Flags
                                Must be of the format '<keyName>:<dataType>'.
       --sort-key string        Optional. Sort key for the DDB table.
                                Must be of the format '<keyName>:<dataType>'.
+RDS Flags
```

Comment:
`copilot storage init -h` shows this as "Aurora Serverless Flags" should we update it here as well?